myapp/spec/rails_helper.rb

```diff
@@ -0,0 +1,52 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
```

Comment:
I recommend adding the gem https://github.com/DatabaseCleaner/database_cleaner to clean the database after each test to make each test totally independent from the others